src/main/java/org/fcrepo/camel/processor/SparqlDescribeProcessor.java

```diff
@@ -46,13 +45,11 @@
      *  @param exchange the current camel message exchange
      */
     public void process(final Exchange exchange) throws IOException {
-
-        final Message in = exchange.getIn();
-        final String subject = ProcessorUtils.getSubjectUri(in);
+        final String subject = getSubjectUri(exchange.getIn()).orElseThrow(() ->
```

Comment:
For this I think it would be best to simply use `ExchangeHelper::getMandatoryHeader` (rather than subclassing).